src/main/kotlin/frc/robot/subsystems/superstructure/ArmPresets.kt

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems.superstructure
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup
+import frc.robot.subsystems.climb.ClimbSubsystem
+import org.ghrobotics.lib.commands.sequential
+import org.ghrobotics.lib.mathematics.units.derived.degrees
+import org.ghrobotics.lib.mathematics.units.inches
+
+object ArmPresets {
+    //Elevator, proximal then wrist (order for commands
+    //TODO Put in real presets!!!!!
+    var hab3 = false
+    var hab2 = false
+                //35 inches is elevator true "0"
+        fun cargoLow(){
+               Superstructure.goToPreset(25.inches, 90.degrees, 10.degrees)
+        }
+        fun cargoMid(){
+            Superstructure.goToPreset(50.inches, 90.degrees, 10.degrees)
+        }
+        fun cargoHigh(){
+            Superstructure.goToPreset(75.inches, 90.degrees, 10.degrees)
+        }
+        fun hatchLow(){
+            Superstructure.goToPreset(25.inches, 90.degrees, 0.degrees)
+        }
+        fun hatchMid(){
+            Superstructure.goToPreset(50.inches, 90.degrees, 0.degrees)
+        }
+        fun hatchHigh(){
+            Superstructure.goToPreset(75.inches, 90.degrees, 0.degrees)
+        }
+        fun stowed(){
+            Superstructure.goToPreset(15.inches, 90.degrees, 0.degrees)
+        }
+        fun hab3Prep(){
+            Superstructure.goToPreset(35.inches, (-5).degrees, 93.degrees) //right
+            hab3 = true
+            hab2 = false
+        }
+        fun hab2Prep(){
+            Superstructure.goToPreset(15.inches, (-5).degrees, 93.degrees) //tune
+            hab3 = false
+            hab2 = true
+
+        }
+        fun habYEET(){
+            if(hab3) {
+                ClimbSubsystem.habClimberStilt.setPosition(20.inches) //change to actual height
+                Superstructure.goToPreset(10.inches, (-5).degrees, 93.degrees) //good angle bad height
+            }
+            if(hab2) {
+                ClimbSubsystem.habClimberStilt.setPosition(20.inches) //change to actual height
+                Superstructure.goToPreset(8.inches, (-5).degrees, 93.degrees) // good angle bad height
```

Comment:
So you set the stilt motor to go up and tell the super structure to go do thing, but the wheels on the back of the intake dont do anything. might want those

Reply:
also, it's not very idiomatic to have a call to `setPosition` outside of a command -- can you make a preset command like proximal and wrist?